src/components/ReportsPage/HeatmapPage/Tables/LevelsDrivenTable.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableRow from '@material-ui/core/TableRow/TableRow';
+import { TableCell, Typography } from '@material-ui/core';
+import { GroupByOption, ScorecardServiceScore } from '../../../../api/types';
+import TableBody from '@material-ui/core/TableBody/TableBody';
+import { EntityRefLink } from '@backstage/plugin-catalog-react';
+import { parseEntityName } from '@backstage/catalog-model';
+import { defaultComponentRefContext } from '../../../../utils/ComponentUtils';
+import {
+  getServicesInLevelsFromScores,
+  StringIndexable,
+} from '../HeatmapUtils';
+import { HeatmapTableHeader } from './HeatmapTableHeader';
+import { LevelsInfoCell } from '../LevelsInfoCell';
+
+interface LevelsDrivenTableProps {
+  levels: string[];
+  groupBy: GroupByOption;
+  data: StringIndexable<ScorecardServiceScore[]>;
+}
+
+export const LevelsDrivenTable = ({
```

Comment:
this is too crazy not to share LOL
![image](https://user-images.githubusercontent.com/2001609/154024118-ed56cc0e-e4a9-4cf3-a457-076ee121d8ee.png)

i have 3000 services locally so it gets to look super wild

Reply:
but very mesmerizing tbh